libs/zedUpload/datastore_http.go

```diff
@@ -79,6 +81,25 @@ func (ep *HttpTransportMethod) WithSrcIPAndProxySelection(localAddr net.IP,
 	return nil
 }
 
+// WithSrcIPAndHTTPSCerts skip verify or append certs for https datastore
+func (ep *HttpTransportMethod) WithSrcIPAndHTTPSCerts(localAddr net.IP, skip bool, certs [][]byte) error {
+	ep.hClient = httpClientSrcIP(localAddr, nil)
+	if ep.hClient != nil {
+		if skip {
+			ep.hClient.Transport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+		} else if len(certs) > 0 {
+			caCertPool := x509.NewCertPool()
+			for _, pem := range certs {
+				if !caCertPool.AppendCertsFromPEM(pem) {
+					return fmt.Errorf("Failed to append datastore certs")
+				}
+			}
```

Comment:
This is fine with me, but then we need to document that the certs in the UI are not added to the base Linux set of root CAs. Thus different than what we do for the proxy where we add the cert(s) to that base set.

Reply:
right. in proxy case, there is a transport.Proxy item and also the proxy cert chain append into the linux certs directory. In this datastore case, the user explicitly want to have this cert chain to verify for this endpoint of https.